sigopt/xgboost/run.py

```diff
@@ -0,0 +1,110 @@
+import time
+import sys
+import platform
+import copy
+
+import xgboost
+# pylint: disable=no-name-in-module
+from xgboost import DMatrix
+
+from ..context import Context
+from ..log_capture import SystemOutputStreamMonitor
+from .. import create_run
+
+DEFAULT_EVALS_NAME = 'Test Set'
+DEFAULT_RUN_OPTIONS = {
+  'log_sys_info': True,
+  'log_stdout': True,
+  'log_stderr': True,
+  'run': None
+}
+XGB_ALIASES = \
+  {
+     'learning_rate': 'eta',
+     'min_split_loss': 'gamma',
+     'reg_lambda': 'lambda',
+     'reg_alpha': 'alpha'
+  }
```

Comment:
Syntax follow the one above.

Reply:
Should I delete this if we don't care about the aliasing anymore

Reply:
ah true. yea